apps/client/src/app/pages/search/search-result/search-result.component.ts

```diff
@@ -11,6 +11,10 @@ import { HtmlToolsService } from '../../../core/tools/html-tools.service';
 })
 export class SearchResultComponent {
 
+  //Bounds for number of a single item to be added to list
+  maxAmount = 999;
```

Comment:
Same as for the list panel maxAmount property (might want to put these inside a global configuration file)